packages/mobile/src/app/saga.ts

```diff
@@ -113,21 +113,25 @@ export interface RemoteFeatureFlags {
 }
 
 export function* appRemoteFeatureFlagSaga() {
-  const remoteFeatureFlagChannel = yield call(appRemoteFeatureFlagChannel)
-  if (!remoteFeatureFlagChannel) {
-    return
-  }
-  try {
-    while (true) {
-      const flags: RemoteFeatureFlags = yield take(remoteFeatureFlagChannel)
-      Logger.info(TAG, 'Updated feature flags', JSON.stringify(flags))
-      yield put(updateFeatureFlags(flags))
-    }
-  } catch (error) {
-    Logger.error(`${TAG}@appRemoteFeatureFlagSaga`, error)
-  } finally {
-    if (yield cancelled()) {
-      remoteFeatureFlagChannel.close()
+  // Refresh feature flags on process start
+  // and every hour afterwards when the app becomes active.
+  // If the app keep getting killed and restarted we
+  // will load the flags more often, but that should be pretty rare.
+  // if that ever becomes a problem we can save it somewhere persistent.
+  let startTime = 0
+
+  while (true) {
+    Logger.debug(TAG, 'appRemoteFeatureFlagSaga in while loop')
+    const action: SetAppState = yield take(Actions.SET_APP_STATE)
```

Comment:
Let's do this last to ensure it refreshes the flag at least once after launching.